assets/js/util.js

```diff
@@ -242,8 +243,19 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function callApy(options, endpoint) {
+    return $.ajax(Object.assign({
+        url: config.APY_URL + endpoint,
+        beforeSend: ajaxSend,
+        contentType: 'application/x-www-form-urlencoded; charset=UTF-8',
+        dataType: 'json',
+        type: (options.data.q.length > THRESHOLD_REQUEST_LENGTH) ? 'POST' : 'GET'
```

Comment:
> Sum up the lengths of keys and values of all the params in requestOptions object.

Not exactly. Only `options.data` gets sent in the URL.

> I am unable to determine ultimately what all parameters will contribute to the GET request packet!

Read into `$.get` and the GET HTTP request.

> Construct the URL similar to something that this snippet from persistChoices function generates? and include what parts of this URL as a part of my request data length? (Eg: only & and =, + kind of stuff? Since the other values have already been considered while traversing the object

I would suspect that there's an easier method with jQuery. If not, then yes, a similar fashion would suffice.